ca_gazebo/launch/apriltag.launch

```diff
@@ -0,0 +1,34 @@
+<launch>
+	<arg name="env" default="apriltags"/>
+
+  	<arg name="paused" value="$(optenv PAUSED false)"/>
+	<arg name="gui" value="$(optenv GUI true)"/>
+	<arg name="debug" value="$(optenv DEBUG false)"/>
+
+	<!-- Launch Gazebo world -->
+	<include file="$(find gazebo_ros)/launch/empty_world.launch">
+		<arg name="debug" value="$(arg debug)" />
+		<arg name="gui" value="$(arg gui)" />
+		<arg name="paused" value="$(arg paused)"/>
+		<arg name="use_sim_time" value="true"/>
+		<arg name="verbose" value="true"/>
+		<arg name="world_name" value="$(find ca_gazebo)/worlds/$(arg env).world"/>
+	</include>
+	
+	<param name="create1/amcl/initial_pose_x" value="2.15"/>
+	<param name="create1/amcl/initial_pose_y" value="-0.2"/>
+	<param name="create1/amcl/initial_pose_a" value="0"/>
+	<param name="create2/amcl/initial_pose_x" value="5"/>
+	<param name="create2/amcl/initial_pose_y" value="1"/>
+	<param name="create2/amcl/initial_pose_a" value="1.5708"/>
+
+
+	<!-- Spawn robot/s -->
```

Comment:
Are these lines necessary? (I'm comparing this launch file with others and they don't have them).

Reply:
Adressed